docs/nydusd.md

```diff
@@ -69,15 +69,14 @@ mount -t virtiofs nydus /mnt
 
 #### Common Fields In Config
 
-``` JSON
+```
 {
   "device": {
     "backend": {
       // localfs | oss | registry
-      "type": "...",
+      "type": "localfs",
       "config": {
-        ...
-        // Access remote storage backend via P2P proxy, for example Dragonfly client address
```

Comment:
Though I am not inclining toward adding such 'fake' comments as it is not Json grammar, we still merge this right now. Hopefully, oneday, we can split that part out from the examples and show a proper Json.

Reply:
Thanks, @changweige, it's a good suggestion, we can put these comments to outside, or support `yaml` or `toml` config file.